src/components/dev-dot-content/mdx-components/mdx-a/index.tsx

```diff
@@ -0,0 +1,7 @@
+import InlineLink from 'components/inline-link'
+
+function MdxA(props) {
+	return <InlineLink {...props} textWeight="medium" />
```

Comment:
`MdxA` is a very light wrapper around `InlineLink`. The consolidated URL adjustments, and a minor tweak within `InlineLink`, are what make this possible.